packages/js/manifests/polywrap/src/formats/polywrap/migrators/0.3.0_to_0.4.0.ts

```diff
@@ -0,0 +1,15 @@
+import { PolywrapManifest as OldManifest } from "../0.3.0";
+import { PolywrapManifest as NewManifest } from "../0.4.0";
+
+import { ILogger } from "@polywrap/logging-js";
+
+export function migrate(migrate: OldManifest, logger?: ILogger): NewManifest {
+  logger?.info(
+    `A new Docs manifest extension is available! Use 'polywrap docs init' to set up documentation for your Wrap.`
```

Comment:
Nice touch!

Reply:
Agreed, nice touch! I was initially thinking we would want to use something like `polywrap manifest init docs` for this, but I can get behind having a `polywrap docs` root command, just like we have for `polywrap build`.

Reply:
I do think we could also suggest the user use `polywrap manifest schema docs` to see what properties are available.

Reply:
I've updated the `polywrap docs init` command to emit a fully commented yaml version of the docs manifest, so no need to suggest the user "read the docs" via the `polywrap manifest schema docs` command.